sdk/appconfiguration/Azure.Data.AppConfiguration/src/AuthenticationPolicy.cs

```diff
@@ -80,7 +80,7 @@ private void AddHeaders(HttpMessage message, string contentHash)
             const string signedHeaders = "date;host;x-ms-content-sha256"; // Semicolon separated header names
 
             var uri = request.Uri.ToUri();
-            var host = uri.Host;
+            var host = uri.Authority;
```

Comment:
Is this in the new feature set that has not been GA'ed or do we need to rethink GA release?

Reply:
There's no urgency in pushing this out, as it does not impact normal usage.  It only impacts environments which use local DNS overrides or other internal redirection, which are a rare edge case.  This came to our attention via #41051 where there's an informal emulator for the service under development and a non-standard port was used.